### PR TITLE
Feat/docs hero build for less

### PR DESCRIPTION
**Pull request by brendan-defi**
**What changed? Why?**
* Created a Hero component in Base Docs
* Refactored the Base Docs index page to use the new Hero component
* Hero component implements Amplitude experiments logic to display different variants to users based on group (treatment/control)

**Notes to reviewers**
* breakpoints at common device sizes: large monitor, laptop, ipad, smartphone

**How has it been tested?**
locally

### CODE REVIEW

File: apps/base-docs/src/components/Hero/TreatmentHero/BarChart/index.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    <div className={styles.chartColumnContainer}>
+      <div className={styles.chartColumnValue}>${columnData.value}</div>
+      <div style={{ height: `${columnHeight}%` }} className={styles[displayType]}>
+        <div className={styles.chartColumnName}>{columnData.name.toUpperCase()}</div>
+      </div>
+    </div>
+  );
+}
+
+const chartData = [
```
Comment (zencephalon) [inline apps/base-docs/src/components/Hero/TreatmentHero/BarChart/index.tsx:50]: In the future it'd be cool to fetch this data. Assuming the experiment goes well these numbers will go out of date.

Comment (brendan-defi) [reply apps/base-docs/src/components/Hero/TreatmentHero/BarChart/index.tsx:50]: Agreed. I've spent a bit of time trying to find a source, let's discuss.

File: apps/base-docs/src/components/Hero/ControlHero/index.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+import HeroButton from '../HeroButton';
+import styles from './styles.module.css';
+
+export default function ControlHero() {
+  return (
+    <header className={styles.heroContainer}>
+      <div className="layout-container">
+        <div className={styles.heroTextContainer}>
+          <h1 className={styles.heroText}>EVERYTHING YOU NEED TO BUILD ONCHAIN</h1>
```
[the remark this answers is not in the document]
Comment (brendan-defi) [reply apps/base-docs/src/components/Hero/ControlHero/index.tsx:11]: It does, I was belt-and-suspenders'ing. I'll change to normal casing here.

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

**Review by zencephalon — APPROVED** [review]

**Pull request merged**